pkg/shipper/shipper.go

```diff
@@ -144,6 +152,8 @@ func (s *Shipper) Sync(ctx context.Context) {
 	// Reset the uploaded slice so we can rebuild it only with blocks that still exist locally.
 	meta.Uploaded = nil
 
+	// TODO(bplotka): If there are no blocks in the system check for WAL dir to ensure we have actually
+	// access to real TSDB dir (!).
```

Comment:
if `sync` errors should we not return the error?

Reply:
Line `163`

Reply:
Good question, but no. I can add comment why.